.github/workflows/build-containers.yml

```
@@ -169,4 +169,40 @@ jobs:
           repository: flowforge/file-server
           username: flowforge
           password: ${{ secrets.DOCKER_HUB_PASSWORD }}
-          readme-filepath: ./helm/file-server/README.md
\ No newline at end of file
+          readme-filepath: ./helm/file-server/README.md
+
+
+  build_device_agent_container:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          repository: 'flowforge/helm'
+          path: 'helm'
+      - name: Docker Meta Data
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          tags: |
+            type=semver,event=tag,pattern={{version}}-2.2.3
+          flavor: |
+            latest=false
```

We should tag the newest build as latest

```suggestion
            latest=true
```